Emby.Server.Implementations/ServerApplicationPaths.cs

```diff
@@ -107,19 +107,19 @@ public class ServerApplicationPaths : BaseApplicationPaths, IServerApplicationPa
         /// <value>The user configuration directory path.</value>
         public string UserConfigurationDirectoryPath => Path.Combine(ConfigurationDirectoryPath, "users");
 
-        public string DefaultTranscodingTempPath => _defaultTranscodingTempPath ?? (_defaultTranscodingTempPath = Path.Combine(ProgramDataPath, "transcoding-temp"));
+        public string DefaultTranscodePath => _defaultTranscodingTempPath ?? (_defaultTranscodingTempPath = Path.Combine(ProgramDataPath, "transcodes"));
```

Comment:
This is the line I wanted to add to update the default directory, which I had started a while ago but hadn't finished yet.

Reply:
I'm sorry but I don't understand what you meant here :)